public/app/features/serviceaccounts/ServiceAccountTable.tsx

```diff
@@ -0,0 +1,227 @@
+      },
+      {
+        id: 'name',
+        header: 'Account',
+        cell: ({ cell: { value }, row: { original } }: Cell<'role'>) => {
+          const href = `/org/serviceaccounts/${original.id}`;
+          const ariaLabel = `Edit service account's ${name} details`;
+          if (!value) {
+            return null;
+          }
+          return isLoading ? (
+            <Skeleton width={100} />
+          ) : (
+            <TextLink href={href} aria-label={ariaLabel} color="primary">
+              {value}
+            </TextLink>
+          );
+        },
+        sortType: 'string',
+      },
+      {
+        id: 'id',
+        header: 'ID',
+        cell: ({ cell: { value }, row: { original } }: Cell<'role'>) => {
+          const href = `/org/serviceaccounts/${original.id}`;
+          const ariaLabel = `Edit service account's ${name} details`;
+          if (!value) {
+            return null;
+          }
+          return isLoading ? (
+            <Skeleton width={100} />
+          ) : (
+            <TextLink href={href} aria-label={ariaLabel} color="secondary">
+              {original.login}
+            </TextLink>
+          );
+        },
+        sortType: 'string',
+      },
+      {
+        id: 'role',
+        header: 'Roles',
+        cell: ({ cell: { value }, row: { original } }: Cell<'role'>) => {
+          const canUpdateRole = contextSrv.hasPermissionInMetadata(AccessControlAction.ServiceAccountsWrite, original);
+          return isLoading ? (
+            <Skeleton width={100} />
+          ) : contextSrv.licensedAccessControlEnabled() ? (
+            displayRolePicker && (
+              <UserRolePicker
+                userId={original.id}
+                orgId={original.orgId}
+                basicRole={value}
+                roles={original.roles || []}
+                onBasicRoleChange={(newRole) => onRoleChange(newRole, original)}
+                roleOptions={roleOptions}
+                basicRoleDisabled={!canUpdateRole}
+                disabled={original.isExternal || original.isDisabled}
+                width={40}
+              />
+            )
+          ) : (
+            <OrgRolePicker
+              aria-label="Role"
+              value={value}
+              disabled={original.isExternal || !canUpdateRole || original.isDisabled}
+              onChange={(newRole) => onRoleChange(newRole, original)}
+            />
+          );
+        },
+      },
+      {
+        id: 'tokens',
+        header: 'Tokens',
+        cell: ({ cell: { value }, row: { original } }: Cell<'role'>) => {
+          const href = `/org/serviceaccounts/${original.id}`;
+          const ariaLabel = `Edit service account's ${name} details`;
+          return isLoading ? (
+            <Skeleton width={100} />
+          ) : (
+            <Stack alignItems="center">
+              <Icon name="key-skeleton-alt"></Icon>
+              <TextLink href={href} aria-label={ariaLabel} color="primary">
```

Comment:
I don't think this needs to be a link, the first three columns should be enough.

Reply:
🤷 I copied it from the original code. It is how it works on `main`. Maybe a question for the **identity team**?